src/integrations/CustomerIO/browser.js

```diff
@@ -11,6 +13,7 @@ class CustomerIO {
     this.analytics = analytics;
     this.siteID = config.siteID;
     this.apiKey = config.apiKey;
+    this.sendPageNameInSDK = config.sendPageNameInSDK;
```

Comment:
add default value

Reply:
default value is being sent from config itself